crypto/rand/drbg_lib.c

```diff
@@ -238,8 +315,16 @@ int RAND_DRBG_reseed(RAND_DRBG *drbg,
 
     if (!ctr_reseed(drbg, entropy, entropylen, adin, adinlen))
         goto end;
+
     drbg->state = DRBG_READY;
-    drbg->reseed_counter = 1;
+    drbg->generate_counter = 0;
+    drbg->reseed_time = time(NULL);
+    if (drbg->reseed_counter > 0) {
+        if (drbg->parent == NULL)
+            drbg->reseed_counter++;
+        else
+            drbg->reseed_counter = drbg->parent->reseed_counter;
```

Comment:
Do we intent to support chains with more than 2 DRBGs where we expect each child to reseed once the parent reseeds?

This could probably also use a line of comment.

Reply:
This is correct. But I think the revised comments in rand_lcl.h should now be sufficient, so this does not have to be restated here again. Moreover since any such comment would have to be repeated in RAND_DRBG_instantiate(), which contains similar code.